src/test/java/org/apache/commons/crypto/cipher/AbstractCipherTest.java

```diff
@@ -251,13 +247,13 @@ private void byteArrayTest(final String transformation, final byte[] key, final
 		final int n = enc.doFinal(input, 0, input.length, temp, 0);
 		final byte[] cipherText = new byte[n];
 		System.arraycopy(temp, 0, cipherText, 0, n);
-		Assert.assertArrayEquals("byte array encryption error.", output, cipherText);
+		assertArrayEquals( output, cipherText);
```

Comment:
You lost the assertion message and the formatting should match the rest of the code.

Reply:
Done